docs/testplan/pfc/PFC_PAUSE_RESPONSE_HEADROOM_README.md

```diff
@@ -0,0 +1,50 @@
+
+SONiC has two lossless priorities: 3 and 4, by default. Note that only the lossless priorities can react to or generate PFC frames. In other words, PFC frames should not have any impact on traffic on lossy priorities. Packets with Differentiated Services Code Point (DSCP) 3 and 4 are mapped to priority 3 and 4, respectively.
+
+### PFC Headroom
+After a device sends a PFC pause frame, it will take some time for the pause frame to arrive at the upstream device and take effect. To absorb packets arriving during this period of time, we need to reserve some buffer space. This reserved buffer space is called "PFC headroom". The PFC Headroom value depends on several factors, including the response time of the sender to react to PFC Pause frames upon receiving one - known as the PFC response time. 
+
+### PFC Response Times
+The IXIA chassis allows PFC response times to be configured manually i.e. create an additional delay between the time an IXIA port receives a PFC Pause Frame with a "pause quanta", and the IXIA port correspondingly responds to the PFC Pause Frame. With the “PFC Pause delay” feature enabled, the IXIA port will wait for `N` quanta before stopping traffic.  The pause time by itself is unchanged.  The “PFC Pause delay” effectively increases the PFC Pause response time. This can be configured by setting the <code>pfc.pfc_delay</code> parameter (corresponding to `N` in our example).  
+
+Put another way, if the IXIA port is configured with a “PFC Delay” of `N` and it receives a PFC pause frame with `Y` quanta, the port will wait for `N` quanta before stopping traffic for `Y` quanta. 
+
+For a "PFC Pause Delay" value that is sufficiently large, the Tx IXIA port takes some additional time to respond to PFC Pause Frames generated by the ingress switch port, and slow down the traffic flow. In the mean time, the Tx IXIA port continues to send traffic at line rate to a route that is already congested. As a result, there is a lot of in-flight packets to capture in the switch's headroom buffer which it fails to capture (due to limited headroom buffer space) and overflows, causing a loss in traffic to the destination port.
```

Comment:
"capture" seems very weird. We typically say no buffer space to "store" or "hold" packets.